workflow_support_scripts/convert_curve_files.py

```diff
@@ -0,0 +1,92 @@
+"""
+Convert RHFE sensor data into the correct format for ISIS
+"""
+
+import os
+from io import open
+import argparse
+from datetime import date
+import json
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+ORIGINAL_FILE_EXTENSION = ".curve"
+OUTPUT_FILE_EXTENSION = ".txt"
+isis_calibration = {"ISIS calibration": {"sensor_type": "RhFe",
+                                         "format_version": "1",
+                                         "conversion_date": date.today(),
+                                         "column1_name": "Temp",
+                                         "column1_units": "K",
+                                         "column2_name": "Measurement",
+                                         "column2_units": "Ohms"}
+                    }
```

Comment:
Could we move static variables such as these into a config file? So maybe a file called 'config.py' that looks similar to:
```
from datetime import date

class Constants:
    """constants used by conversion scripts"""
    WIKIURL = "https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Calibration-Files"
    ENCODING = "utf-8"


class FileTypes:
    ORIGINAL_CURVE_FILE_EXTENSION = ".curve"
    ORIGINAL_DAT_FILE_EXTENSION = ".dat"
    OUTPUT_FILE_EXTENSION = ".txt"

class ISISCalibration:
    isis_calibration = {
        "ISIS calibration": 
        {
            "sensor_type": "RhFe",
            "format_version": "1",
            "conversion_date": date.today(),
            "column1_name": "Temp",
            "column1_units": "K",
            "column2_name": "Measurement",
            "column2_units": "Ohms"
        }
    }
```